tests/test_project.py

```diff
@@ -169,6 +204,13 @@ def test_save_project_artifact_multi_experiment(tmp_path):
 
     # Check that the first experiment artifact was not overwritten
     fs = fsspec.filesystem("file")
+
+    print(f'\nfirst, created_at: {project["my-first-experiment"].created_at}\n')
+    print(f'\nfirst, fs.created: {fs.created(location / project["my-first-experiment"].path / "features.json")}\n')
+    
+    print(f'\nsecond, created_at: {reload_project["my-second-experiment"].created_at}\n')
+    print(f'\nsecond, fs.created: {fs.created(location / reload_project["my-first-experiment"].path / "features.json")}\n')

```

Comment:
I'm guessing this is from your timezone issue, right?

Reply:
Ah yes. My bad, forgot to delete these debug print lines. I'll take them out